.github/workflows/quick_tests.yml

```diff
@@ -23,8 +23,26 @@ jobs:
                        - name: install dependecies
                          run: |
                            pip install jaxlib
-                           pip install plum-dispatch
                        - name: install pysages
                          run: python setup.py install
                        - name: test pysages installation
-                         run : python3 -c "import pysages"
+                         run : |
+                           cd /
+                           python3 -c "import pysages"
+        install2:
+                runs-on: ubuntu-20.04
+                name: "Testing installation of pysages with pip"
+                steps:
+                       - uses: actions/checkout@v2
+                       - uses: actions/setup-python@v2
+                         with:
+                             python-version: '3.x'
+                       - name: install dependecies
+                         run: |
+                           pip install jaxlib
+                       - name: install pysages
+                         run: pip install .
```

Comment:
I think it needs to be run with `-e`

Reply:
`-e` is for editable build.
That's a feature for developing, it has to work without it too.